usr/share/rear/conf/default.conf

```diff
@@ -1553,6 +1553,20 @@ CLONE_ALL_USERS_GROUPS="true"
 # Therefore special users and groups can be added via CLONE_USERS and CLONE_GROUPS
 # or even CLONE_ALL_USERS_GROUPS="all" can be specified.
 
+# A terminal login password as a salted hash.
+# If empty, a root login into the ReaR recovery system via the system console or a serial terminal
+# is possible without being asked for a password.
+# To generate a password hash,
+# 1. run
+#    - 'openssl passwd -6', if openssl 1.1.1 or newer is available, or
+#    - 'openssl passwd -1', otherwise (see https://github.com/rear/rear/pull/2455#discussion_r453613086 for information
+#      on the implications of its cryptographic weakness),
+# 2. copy its entire output line and insert it between the single quotes of the following line:
+#        { TTY_ROOT_PASSWORD=''; } 2>/dev/null
+#    NOTE: stderr is redirected in the above line to avoid exposing the password hash
+#          in the log file when ReaR runs in debugscript mode.
+TTY_ROOT_PASSWORD=''
```

Comment:
Related to not expose *_ROOT_PASSWORD values
see what I did in https://github.com/rear/rear/pull/2541 via
https://github.com/rear/rear/pull/2541/commits/52928510e6788f683593db095453435131d36bc3
```
Set SSH_ROOT_PASSWORD to a default value
only if not already set so that the user can set it e.g. with
export SSH_ROOT_PASSWORD=my_recovery_system_root_password
before he calls "rear ..." so there is no need
to have SSH_ROOT_PASSWORD in local.conf.
```
for usr/share/rear/conf/default.conf
```
# Avoid that the SSH_ROOT_PASSWORD value is shown when usr/sbin/rear was called with 'set -x'
# for debugging usr/sbin/rear cf. https://github.com/rear/rear/issues/2144#issuecomment-493908133
# In debugscript mode only scripts sourced by the Source function in lib/framework-functions.sh
# are run with 'set -x' but default.conf is sourced by usr/sbin/rear directly.
# See the comment of the UserInput function in lib/_input-output-functions.sh
# how to keep things confidential when usr/sbin/rear is run in debugscript mode
# ('2>/dev/null' should be sufficient here because 'test' does not output on stdout).
# SSH_ROOT_PASSWORD is set to a default value here only
# if not already set so that the user can set it also like
#   export SSH_ROOT_PASSWORD='my_recovery_system_root_password'
# directly before he calls "rear ...":
{ test "$SSH_ROOT_PASSWORD" ; } 2>/dev/null || SSH_ROOT_PASSWORD=''
```